src/variants/lambdaregistry/storage.mligo

```diff
@@ -58,10 +58,29 @@ type token_transfer =
   ; transfer_list : transfer_item list
   }
 
+type legacy_transfer_target =
+  [@layout:comb]
+  { to : address
+  ; value : nat
+  }
+
+type legacy_transfer =
+  [@layout:comb]
+  { from : address
+  ; target : legacy_transfer_target
+  }
+
+type legacy_token_transfer =
+  [@layout:comb]
+  { contract_address : address
+  ; transfer : legacy_transfer
+  }
+
```

Comment:
I'm not very familiar with LIGO, why is this duplicated from `common/types.mligo`? Or am I missing something?

Reply:
I may be wrong, but IIRC this is forced by the combination of:
1. the layout of the variants
2. the fact that LIGO is compiled top to bottom and `include` really just put the included code there

I'm going off my memory tho, so I'd like @sras to confirm and add more details